Code/Editor/EditorEngineProcessFramework/EngineProcess/EngineProcessDocumentContext.h

```diff
@@ -24,17 +42,17 @@ class EZ_EDITORENGINEPROCESSFRAMEWORK_DLL ezEngineProcessDocumentContext : publi
   EZ_ADD_DYNAMIC_REFLECTION(ezEngineProcessDocumentContext, ezReflectedClass);
 
 public:
-  ezEngineProcessDocumentContext();
+  ezEngineProcessDocumentContext(ezBitflags<ezEngineProcessDocumentContextFlags> flags = ezEngineProcessDocumentContextFlags::Default);
   virtual ~ezEngineProcessDocumentContext();
 
-  void Initialize(const ezUuid& DocumentGuid, ezEngineProcessCommunicationChannel* pIPC);
+  virtual void Initialize(const ezUuid& DocumentGuid, const ezVariant& metaData, ezEngineProcessCommunicationChannel* pIPC);
```

Comment:
make sure this isn't called from constructors, because you know how they like virtual functions

Reply:
Not possible as these are created via reflection and the void ctor. Should be safe.